metricbeat/include/list.go

```diff
@@ -25,6 +25,7 @@ import (
 	_ "github.com/elastic/beats/metricbeat/module/system/diskio"
 	_ "github.com/elastic/beats/metricbeat/module/system/filesystem"
 	_ "github.com/elastic/beats/metricbeat/module/system/fsstat"
+	_ "github.com/elastic/beats/metricbeat/module/system/load"
```

Comment:
why was this changed? switch forth / back between branches? `make collect` should clean it up.